msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -14,7 +14,7 @@ The resource-specific consent (RSC) permissions model, originally developed for
 
 > [!NOTE]
 >
-> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md).
+> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md) and will only be enabled after a new installation into a chat.
```

Comment:
Some confusion between Quan and I - we were both working on similar doc updates. Sharing my suggestions and pending changes here so we can consolidate everything in this PR:

1. Suggest move [note] section down, above the [Important] section
2. Update notes with the following bullets:
```
-  The ability for bots to receive all messages in chats using ChatMessage.Read.Chat is available only in [public developer preview for Teams](https://file+.vscode-resource.vscode-cdn.net/c%3A/Code/msteams-docs/msteams-platform/resources/dev-preview/developer-preview-intro.md).
- Bots in apps that have previously defined ChatMessage.Read.Chat permission for Graph scenarios will continue to only receive @mentioned messages in conversations where they are currently install. Bots will begin receiving all messages in chats upon re-installations or new installations.
- Developers with apps that contain a bot and has ChatMessage.Read.Chat defined in the manifest need to validate the new behavior and modify the app if needed
```
3. Add sample code to identify messages with bot mentioned above [Note]

Screen shot of pending changes in this section:
![image](https://user-images.githubusercontent.com/56136707/213590059-24e07eca-67fa-41a1-8c24-2a4635cdeaa5.png)